.travis.yml

```diff
@@ -6,6 +6,9 @@ language: node_js
 node_js:
   - "10"
 
+before_script:
+  - curl https://v4.ifconfig.co/
```

Comment:
What’s the purpose of this?

Reply:
So that we get the machine's IP for the SP guys.